src/types/Boolean.ts

```diff
@@ -0,0 +1,14 @@
+import * as t from 'io-ts'
+
+export interface Boolean extends t.Type<string, string, t.mixed> {}
+
+export const Boolean: Boolean = new t.Type<string, string, string>(
+    'Boolean',
+    t.string.is,
+    (s, c) => {
+        return (s === 'true' || s === 'false')  ? t.success(s) : t.failure(s, c)
```

Comment:
Do you always want lower case?

Reply:
Most of the cases converter from string to boolean works like this:
```
const boolValue = stringValue === 'true'
```
Since `Boolean('false')` returns true 🤷‍♂️